modules/govuk_jenkins/templates/jobs/copy_data_from_staging_to_aws.yaml.erb

```diff
@@ -96,10 +96,13 @@
             description: 'Choose the thing to sync. All is the default, but some jobs may run but not do anything due to your config for the destination environment.'
             choices:
                 - all
+                - attachments
+                - assets
                 - elasticsearch-rummager
                 - mongo-api
                 - mongo-normal
                 - mongo-router
+                - mongo-pp
```

Comment:
We don’t need to sync mongo-pp because Performance Platform (pp) is staying on Carrenza/PaaS.

Reply:
Maybe this is another story to remove it from AWS if it is in staging. The database shouldn’t have anywhere to sync to as we shouldn’t have any `performance-mongo` machines.